psi4/src/psi4/libsapt_solver/exch-ind30.cc

```diff
@@ -314,5 +318,263 @@ double SAPT2p3::exch_ind30_3(double **sBS) {
 
     return (energy);
 }
+
+// Compute the nonapproximated third-order exchange-induction //
+// Konrad Patkowski, based on Jonathan Waldrop's Psi4NumPy code //
+// September 2021 //
+void SAPT2p3::sinf_e30ind() {
+
+    if (print_) {
+        outfile->Printf("  ==> Nonapproximated third-order induction <==\n\n");
+    }
+
+    // => Sizing <= //
+
+    int nn = basisset_->nbf();
+    int na = noccA_;
+    int nb = noccB_;
+    int nr = nvirA_;
+    int ns = nvirB_;
+
+    int nT = 1;
```

Comment:
I don't think this threading variable is needed

Reply:
It's not. I removed it.